README.md

```diff
@@ -64,7 +64,7 @@ and access the project handle:
 You can test this package by executing:
 
 ```bash
-$ python -m unittest discover tests/
+$ python -m pytest tests/

```

Comment:
No, using `python -m pytest` protects us against any issues where the location where python installs it's CLI entry points is not on the PATH (so it can't find `pytest`). It's not generally an issue, but is somewhat OS dependent and we've run into it before on Linux machines.